run_route_scripts/run_with_server.py

```diff
@@ -93,7 +92,7 @@ def make_request(post_body):
   parser.add_argument('--test-file', type=str, help='The file with the test requests', required=True)
   parser.add_argument('--url', type=str, help='The url to which you want to POST the request bodies', default='http://localhost:8002/route')
   parser.add_argument('--output-dir', type=str, help='The directory in which to place the result of each request')
-  parser.add_argument('--concurrency', type=int, help='The number of processes to use to make requests', default=psutil.cpu_count(logical=True))
+  parser.add_argument('--concurrency', type=int, help='The number of processes to use to make requests', default=multiprocessing.cpu_count())
```

Comment:
was this something mac specific or?

Reply:
No. This happened on Linux as well. I didn't have `psutil` installed so this script was failing. Since `multiprocessing`  has the `cpu_count` I thought of using that instead of `psutil` since its already in use here.

Reply:
i mean you can just pip install it but sure, one less thing seems better (i cant remember why i used that in the first place, maybe one works better under docker or something...)

Reply:
yeah 1 less package to install. I _ think_ this should work inside docker too.